CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+## 15.5.2
+
+_released `2024-02-05`_
+
+### cli
+
+- :wrench: Updated the "custom-auth" sample to avoid a warning when running zapier validate ([#724](https://github.com/zapier/zapier-platform/pull/724))
+
+- :bug: Fixed bug in `zapier convert` that crashes due to syntax error in user's code and should not replace `source` in sample ([#730](https://github.com/zapier/zapier-platform/pull/730))
+
+- :wrench: Improved handling of broken symlinks while copying files to temp directory during the build process ([#737](https://github.com/zapier/zapier-platform/pull/737))
```

Comment:
I think this is a bug fix rather than a refactor?

```suggestion
- :bug: Improved handling of broken symlinks while copying files to temp directory during the build process ([#737](https://github.com/zapier/zapier-platform/pull/737))
```